canopen/sdo/client.py

```diff
@@ -36,9 +162,25 @@ def __init__(self, rx_cobid, tx_cobid, od):
         :param canopen.ObjectDictionary od:
             Object Dictionary to use for communication
         """
-        SdoBase.__init__(self, rx_cobid, tx_cobid, od)
+        self.rx_cobid = rx_cobid
+        self.tx_cobid = tx_cobid
+        self.network = None
+        self.node = node
+        self.od = node.object_dictionary
         self.responses = queue.Queue()
 
+    def __getitem__(self, index):
+        return DataProxy(self, self.node.object_dictionary[index])
```

Comment:
Can't you just use `self.od[index]` here and remove `self.node`?

Reply:
Absolutely